src/routes/controllers/ordersController/createOrder.js

```diff
@@ -0,0 +1,28 @@
+import order from '../../../models/order';
+import OrderModel from '../../../models/order';
+
+const createOrder = async (req, res) => {
+  try {
+    const user = req.user._id;
+
+    const { shippingAddress, billingAddress, totalAmount, productsPurchased } =
```

Comment:
you should add some validation using Joi. take a look at this for reference 

https://github.com/saifulislamdev/sylica-backend/blob/main/src/routes/controllers/productsControllers/createProduct.js